### PR TITLE
familyView.php UI update

### PR DESCRIPTION
**Pull request by selbyheyman**
- childrenInAccount.php getId() fixed
- familyView.php updated

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -153,6 +153,12 @@
                         <img src="images/person-search.svg">
                         <span>Find Family Account</span>
                 </div>
+
```
Comment (aswann2) [inline index.php:156]: This portion of the code needs to be removed. This button is on the staff dashboard, and this does not apply to the staff. When a staff views a family profile after searching for one, it still pulls up the newer format which is good though.

**Review by aswann2 — CHANGES_REQUESTED** [review]

File: childrenInAccount.php [hunk 2]
```diff
@@ -55,13 +55,13 @@
                         <tbody class="standout">';
                         foreach($children as $acct){
                             echo '<tr>';
-                            echo '<td><a href=childAccount.php?id=' . $acct->getID() . '>' . $acct->getID() . '</a></td>';
```
Comment (itmacleod1995) [inline childrenInAccount.php:58]: I think this should break? There is no getId() function in children.php, only getID()

File: index.php (hunk 1 above)
Comment (selbyheyman) [reply index.php:156]: oops sorry i was trying to test it when I couldn't log into family account, I'll removed it

File: childrenInAccount.php (hunk 2 above)
Comment (selbyheyman) [reply childrenInAccount.php:58]: you're right, I'll change it back

**Pull request closed without merging**